hack/go-lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if [ "$IS_CONTAINER" != "" ]; then
+  golint -set_exit_status installer/...
+else
+  docker run --rm --env IS_CONTAINER='TRUE' -v "$PWD":/go/src/github.com/openshift/installer -w /go/src/github.com/openshift/installer --entrypoint sh quay.io/coreos/golang-testing /go/src/github.com/openshift/installer/hack/go-lint.sh
```

Comment:
nit: you're setting the working directory explicitly with `-w`, so you should be able to use `./hack/go-lint.sh` as the final argument instead of the absolute path you have now.  That would also more closely match the pattern used by our existing hack scripts:

```console
$ git describe --always --abbrev=0
5ac7cfb476972284e7dd25bc14d8d82442f439bc
$ grep 'docker.*hack/' hack/*.sh
hack/gotest.sh:  docker run -e IS_CONTAINER='TRUE' -v "$PWD":/go/src/github.com/openshift/installer -w /go/src/github.com/openshift/installer quay.io/coreos/golang-testing ./hack/gotest.sh
hack/go-vet.sh:    docker run --rm --env IS_CONTAINER='TRUE' -v "$PWD":/go/src/github.com/openshift/installer -w /go/src/github.com/openshift/installer quay.io/coreos/golang-testing ./hack/go-vet.sh "$dir"
hack/shellcheck.sh:  docker run -e IS_CONTAINER='TRUE' --rm -v "$(pwd)":/workdir:ro --entrypoint sh quay.io/coreos/shellcheck-alpine:v0.5.0 /workdir/hack/shellcheck.sh /workdir;
hack/test-bazel-build-tarball.sh:  docker run -e IS_CONTAINER='TRUE' --rm -v "$PWD":"$PWD" -v /tmp:/tmp:rw -w "$PWD" quay.io/coreos/tectonic-builder:bazel-v0.3 ./hack/test-bazel-build-tarball.sh
hack/tf-fmt.sh:  docker run -e IS_CONTAINER='TRUE' --rm -v "$PWD":"$PWD":ro -v /tmp:/tmp:rw -w "$PWD" quay.io/coreos/terraform-alpine:v0.11.7 ./hack/tf-fmt.sh
hack/tf-lint.sh:  docker run -t --rm -v "$(pwd)":/data --env IS_CONTAINER='TRUE' --entrypoint sh quay.io/coreos/tflint ./hack/tf-lint.sh
hack/yaml-lint.sh:  docker run -t --rm -v "$(pwd)":/workdir --env IS_CONTAINER='TRUE' --entrypoint sh quay.io/coreos/yamllint ./hack/yaml-lint.sh
```